execute/plugin.go

```diff
@@ -345,6 +369,10 @@ func (p *Plugin) Outcome(
 		return commitReports[i].Timestamp.Before(commitReports[j].Timestamp)
 	})
 
+	p.lggr.Debugw(
+		fmt.Sprintf("[oracle %d] exec outcome: commit reports", p.reportingCfg.OracleID),
```

Comment:
Why use sprintf instead of a key/value?

Reply:
Having the oracle ID in the log message is much quicker to parse than having it in the fields; so I want a formatted string as the log message but still want to JSON encode the log fields.